package.json

```diff
@@ -4,13 +4,22 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
```

Comment:
you've copied over these commands from the ui repo, however this isn't a react app, react-scripts isn't installed and it doesn't make sense to run these. 

there's not really any reason to have a build script for this repo (for now) so can probs go without `build` in ci